### ISSUE TITLE
SlevomatCodingStandard.Variables.UnusedVariable.UnusedVariable - false positive with operation in condition

### ISSUE THREAD

**Issue by czjvic** [slevomat/coding-standard#1096]
I found false positive, when I use increase operator before variable `++$a` ... In case of` $a++` it works as expected (no error reported) 
```php
function foo() {
    $a = 1;
    for ($i = 1; $i < 1000; $i++) {
        if (++$a > 5) {
            return;
        }
    }
}
```
```
 Unused variable $a.
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/c84eb03ceb70d16f535751542c61dae17af057c9
